manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: vatis-project.vAtis
+PackageVersion: 4.1.0-beta.1
+PackageLocale: en-US
+Publisher: vATIS for VATSIM
+PublisherUrl: https://github.com/vatis-project
+PublisherSupportUrl: https://github.com/vatis-project/vatis-archive/issues
+PackageName: vATIS
+PackageUrl: https://github.com/vatis-project/vatis-archive
+License: GPL-3.0
+LicenseUrl: https://github.com/vatis-project/vatis-archive/blob/HEAD/LICENSE
```

```suggestion
LicenseUrl: https://github.com/vatis-project/vatis/blob/main/LICENSE
```

(See previous comments)